.github/workflows/protobuf.yaml

```diff
@@ -54,7 +54,5 @@ jobs:
           `find ./after/target/debug/build/*/output`
           | xargs cat > ./after.binpb
       - uses: bufbuild/buf-setup-action@v1
```

Comment:
Off-topic: Would setting `github_token: ${{ github.token }}` work for this action in our setup? Seems that the check [fails with rate-limiting errors](https://github.com/matter-labs/zksync-era/actions/runs/8596293564/job/23552565599?pr=1584#step:9:20) sometimes.

Reply:
ack, added